src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Twig/WysiwygExtension.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Copyright 2019 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class WysiwygExtension extends AbstractExtension
+{
+
+    private static $allowedTags = '<p><em><strong><span><h1><h2><h3><h4><h5><h6><ul><ol><li><a><sup><sub><code><blockquote>';
```

Comment:
Good idea to make a custom extension! I'd love to see the allowed tags being configurable in the parameters.json or elsewhere. If we keep them here, please update the readme to instruct users of this feature. Must we consider attribute sanitization? As this is another valid attack vector in this scenario.

For example we now allow the use of span tags. They are filled with inline css to create strikethroughts and underlines. A possible attacker can now also run javascript from the attribute as these are not sanitized.

Reply:
I didn't found it that valuable because it won't be easily configurable due to the Tinymce configuration itself.